v3.3/glfw/go.mod

```diff
@@ -1,3 +1,3 @@
-module github.com/go-gl/glfw/v3.3/glfw
+module github.com/tracer8086/glfw/v3.3/glfw
```

Comment:
`s/tracer8086/go-gl`

Reply:
Should I add it after `module` directive?

Reply:
replace `tracer8086` with `go-gl` ;)